src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars--0 _df">
+        <div class="stars__star"></div>
```

Comment:
Use more obvious class names
```suggestion
    <div class="container">
      <div class="stars stars--0">
        <div class="stars__star"></div>
```